src/main/java/org/jabref/gui/importer/actions/SearchGroupsMigrationAction.java

```diff
@@ -24,11 +23,17 @@ public class SearchGroupsMigrationAction implements GUIPostOpenAction {
     // We cannot have this constant in `Version.java` because of recursion errors
     // Thus, we keep it here, because it is (currently) used only in the context of groups migration.
     public static final Version VERSION_6_0_ALPHA = Version.parse("6.0-alpha");
+    public static final Version VERSION_6_0_ALPHA_1 = Version.parse("6.0-alpha_1");
```

Comment:
there has been no alpha release yet, its all just CI build yet. So i doubt its correct to introduce another ALPHA iteration step here... @koppor ?

Reply:
We kept the old version and introduced a new one to ask users who tested the alpha version and migrated their search syntax to Lucene to revert to their backup, as there is no migration from Lucene back to the old syntax.

Reply:
ok i see. But we should do this now for our power users but remove / streamline it before the real alpha release or so. Otherweise everything will be overpolluted with these horrible migration heuristics

Reply:
Will be streamlined in the beta version. If the format is kept, we will rewrite `6.0-alpha_1`to `6.0-beta` in the beta version and to `6.0` in the final release and drop reading capabilities for `-alpha_1`.